package.json

```diff
@@ -1,9 +1,6 @@
 {
   "name": "root",
```

Comment:
we should pin pnpm with Volta in each package.json

in the root package.json we should additionally error for yarn and npm usage, but we don't want to do this in the others as it would break consumers.

```
"engines": {
    "yarn": "use pnpm",
    "npm": "use pnpm",
    "pnpm": "x.y.z"
  },
  "volta": {
    "node": "16.16.0",
    "pnpm": "x.y.z"
  }
    ```